README.md

```diff
@@ -197,9 +207,18 @@ Returns the Peer ID's `id` as a buffer.
 <Buffer 12 20 d6 24 39 98 f2 fc 56 34 3a d7 ed 03 42 ab 78 86 a4 eb 18 d7 36 f1 b6 7d 44 b3 7f cc 81 e0 f3 9f>
 ```
 
+
+### `toString()`
```

Comment:
Personally I would add `toCID` and have `toString` as `() => this.toCID().toString()`.

This creates a symmetric API `createFromCID`/`toCID` (although "create" is superfluous IMHO, but meh, maybe a refactor for another day) and a nice sensible representation as a string that also enables `${peerId}` or `peerID + ''` without having to explicitly call `toString`.

Reply:
I think there is no `toCID` because there was not `toMultihash`. 
We wanted to keep API surface small, liberal in inputs (accept CID object), but strict in outputs (just basic types: `PeerID`, `String` and `Buffer`)

Reply:
I agree in keeping this in the context of this PR. We can iterate on this in a new PR if it justifies